source/faq.rst

```diff
@@ -21,4 +21,11 @@ points to assets packaged within the experiment (e.g. an image, an additional
 ``resource:`` protocol handler.
 
 
+Why is my experiment ``undefined`` on Beta and Release?
+-------------------------------------------------------
+
+On Nightly and Dev Edition, you can install unsigned experiments if you flip two preferences: ``xpinstall.signatures.required`` and ``extensions.legacy.enabled``.
```

Comment:
And also on unbranded builds from CI: https://wiki.mozilla.org/Add-ons/Extension_Signing#Unbranded_Builds